src/signature/SigningAccount.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+// Copyright 2017 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "xdr/Stellar-SCP.h"
+#include "xdr/Stellar-ledger.h"
+
+namespace stellar
+{
+
+class AccountFrame;
+
+struct SigningAccount
+{
+    AccountID accountID;
+    xdr::xvector<Signer, 20> signers {};
```

Comment:
do not specify `20` here, it's already taken care of by other objects

Reply:
I I remove 20 form here, I'll get a compilation error in assignment from accountFrame.getAccount().signers